config/default/common/config/metadata/stories/clouds/step011.md

```diff
@@ -0,0 +1,3 @@
+In contrast, the linear lines in this false color (VIIRS Bands M11-I2-I1) image flowing from north to south are cloud streets. The orientation of the cloud streets (clouds are white) show that strong, cold winds were blowing from north to south. Cloud streets form when columns of heated air, thermals, rise in the atmosphere and carry heat away from the sea surface. This moist air rises until it hits a warmer air layer (a temperature inversion), the inversion causes the rising thermals to roll over on themselves, forming parallel cylinders of rotating air. On the upward side of the cylinders, water vapor condenses and forms clouds. Along the downnward side, where the air is descending, the skies are clear.
```

Comment:
```suggestion
In contrast, the linear lines in this false color (VIIRS Bands M11-I2-I1) image flowing from north to south are cloud streets. The orientation of the cloud streets (clouds are white) show that strong, cold winds were blowing from north to south. Cloud streets form when columns of heated air, thermals, rise in the atmosphere and carry heat away from the sea surface. This moist air rises until it hits a warmer air layer (a temperature inversion); this inversion causes the rising thermals to roll over on themselves, forming parallel cylinders of rotating air. On the upward side of the cylinders, water vapor condenses and forms clouds. Along the downward side, where the air is descending, the skies are clear.
```

Reply:
can't tell what's different here but i'll commit it

Reply:
Thanks for the trust  :)  

I just broke up this sentence a bit: "...(a temperature inversion); this..."